TouchPortalAPI/sdk_tools.py

```diff
@@ -459,7 +463,7 @@ def validateDefinitionFile(file:Union[str, TextIO]):
 
 ## CLI handlers
 
-def _generateDefinition(script, output_path, indent, skip_invalid:bool=False):
+def _generateDefinition(script, output_path, indent, skip_invalid:bool=False, showMessage=True):
```

Comment:
I thought we came up with a better method than this "showMessage" hack...?

Reply:
Yeah... Sorry I forgot to remove it. Should be removed now.